content/module-reference/utility-modules/shared/tagging.md

```diff
@@ -125,8 +125,12 @@ The "edit tag..." operation allows you to change the name of a tag, though you c
 - A tag set as “category” is not exported in `XMP-dc Subject`. However it is exported in `XMP-lr Hierarchical Subject` as this XMP metadata holds the organization of your tags.
 
 - “synonyms” enrich the tag information and are mainly used to assist search engines. For example “juvenile”, “kid” or “youth” can be set as synonyms of “child”. Synonyms can also be used to translate tag names to other languages.
- 
-The "rename path..." operation is only available in the _tree_ view mode, and it shows the number of tagged images which would be impacted by a change to the name of this tag or category. The rename path window lets the user change the full path of tag, including the catagories to which it belongs (categories can be specified using the pipe "|" symbol). This operation is powerful, but please take care as it can have a significant impact on the metadata of your images. The operation is aborted if the requested change causes a conflict with an existing tag.
+
+The "rename path..." operation is only available in the _tree_ view mode, and it shows the number of tagged images which would be impacted by a change to the name of this tag or category. The rename path window lets the user change the full path of tag, including the categories to which it belongs (categories can be specified using the pipe "|" symbol). This operation is powerful, but please take care as it can have a significant impact on the metadata of your images. The operation is aborted if the requested change causes a conflict with an existing tag.
+
+A quick way to organize the tag structure is the drag and drop. In the _tree_ view mode, you can drag whatever tag (node or leave) and drop it on top of another one (node or leave). The first one and its descendants if any become descendants of the second one. Dragging over a node automatically opens the node.
```

Comment:
>A quick way to organize the tag structure is the drag and drop.

should be: A quick way to organize the tag structure is to drag and drop the tags

> you can drag whatever tag

should be: you can drag any tag

Additionally, what is a *leave* tag?

> The first one and its descendants if any become descendants of the second one.

should be: The first tag and its descendants, if any, become descendants of the second tag.
.

Reply:
> Additionally, what is a leave tag?

In a tree we have the nodes (every element part of the path) which can be closed or opened, and the leaves, the extremities of the paths.

Reply:
It should've been "leaf" tag then ;)

Reply:
You are right ... :confused: